pkg/workceptor/kubernetes.go

```diff
@@ -557,6 +581,17 @@ func (kw *kubeUnit) connectToKube() error {
 	return nil
 }
 
+func readFileToString(filename string) (string, error) {
+	if filename == "" {
```

Comment:
In the unlikely event that some future person stumbles across this function and uses it for something, the "return an empty result if given an empty filename" behavior is pretty unexpected.  I think we should at least call it out with a comment.

Reply:
added a comment